### PR TITLE
CondContain - Fixes

### PR DESCRIPTION
**Pull request by ShaneBeee**
### Description
- Fix issue where cant check if an inventory contains an item stored in a variable

---
**Target Minecraft Versions:** all
**Requirements:** none
**Related Issues:** #2045

### CODE REVIEW

**Comment by Pikachu920** [comment]
can you use a more standard else formatting?

**Comment by ShaneBeee** [comment]
> can you use a more standard else formatting?

I tried to and it kept giving me return warnings, when I finally got it to not give me a warning, I had what looked like too many if statements, Then it suggested to simplify it (which is how this happened)
But if you would like I can expand it to more statements

File: src/main/java/ch/njol/skript/conditions/CondContains.java [hunk 1]
```diff
@@ -115,8 +116,14 @@ public boolean check(final Event e) {
 					} else {
 						if (container instanceof Inventory) {
 							final Inventory invi = (Inventory) container;
-							return items.check(e, (Checker<Object>) type
-									-> type instanceof ItemType && ((ItemType) type).isContainedIn(invi));
+							return items.check(e, (Checker<Object>) type -> {
+								if (type instanceof ItemType && ((ItemType) type).isContainedIn(invi)) {
```
Comment (FranKusmiruk) [inline src/main/java/ch/njol/skript/conditions/CondContains.java]: I'd rather have the following for readability's sake but if it happens to cause issues, it's fine the way it is now.
```suggestion
if (type instanceof ItemType)
	return ((ItemType) type).isContainedIn(invi);
else if (type instanceof ItemStack)
	return invi.contains((ItemStack) type);
```

Comment (ShaneBeee) [reply src/main/java/ch/njol/skript/conditions/CondContains.java]: For sure, Ill do some changes and test it, and if it works all fine Ill push

Comment (ShaneBeee) [reply src/main/java/ch/njol/skript/conditions/CondContains.java]: Okay I made the changes, tested, everything seems to work great 👍

**Pull request merged**